### ISSUE TITLE
Expose CRUD endpoints for feed groups and applications

### ISSUE THREAD

**Issue by davidsbond** [GetStream/stream-go2#81]
Allowing users to programatically create feed groups and applications via an API would open the door for some powerful tooling. Personally, I'd like to be able to use terraform to manage/maintain my feed groups rather than relying on the user-interface

If the endpoints for creating/updating/deleting feed groups and applications was exposed, you could easily build a terraform provider that could automate this

**Comment by davidsbond** [comment]
Having a little snoop around, I can see when creating an app a call is made to:

```
POST https://getstream.io/dashboard/api/organization/{org_id}/app/
```

Similarly, feed groups can be created using:

```
POST https://getstream.io/dashboard/api/app/{app_id}/feed/
```

**Comment by ferhatelmas** [comment]
Exactly, totally agreed. It's a work in progress to make it available in the API.

**Comment by amin3mej** [comment]
Any good news?

**Comment by ferhatelmas** [comment]
@amin3mej We're working on some changes in dashboard and in accordance, changing its backing API. Dashboard will ship first by using old API (nearly done) then we will adapt the API and leverage it (in to be soon shipped dashboard and SDKs). Right now, all I can say is it's closer and stay tuned.

**Comment by fr3fou** [comment]
Any updates? I'd _love_ to use Terraform for managing this as well
